inbc-program/inbc/parser/source_app_parser.py

```diff
@@ -47,13 +56,16 @@ def application_remove(args: argparse.Namespace) -> str:
     manifest = ('<?xml version="1.0" encoding="utf-8"?>' +
                 '<manifest><type>source</type>' +
                 '<applicationSource>' +
-                '<remove><gpg>'
-                f'{create_xml_tag(arguments, "keyname")}' +
-                '</gpg><repo>' +
-                f'{create_xml_tag(arguments, "filename")}'
-                '</repo>'
-                '</remove></applicationSource>' +
-                '</manifest>')
+                '<remove>')
```

Comment:
**Suggestion:** Refactor the XML generation into a separate function to reduce code duplication and improve maintainability. [maintainability]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    def generate_manifest(tag_name, content):
        return (f'<?xml version="1.0" encoding="utf-8"?>'
                f'<manifest><type>source</type>'
                f'<applicationSource>'
                f'<{tag_name}>{content}</{tag_name}></applicationSource>'
                f'</manifest>')

    # Usage example:
    manifest = generate_manifest('remove', content)
```